pages.ne/common/ls.md

```diff
@@ -0,0 +1,36 @@
+# ls
+
+> एउटा डिरेक्टोरीमा भएका सबै वस्तुहरुको सुची देखाउँछ। .
```

Comment:
Sorry, the "." symbol was redundant. I apologize for not reviewing properly. In Nepali "।" marks a completion of a declarative sentence.

Reply:
> Sorry, the "." symbol was redundant. I apologize for not reviewing properly. In Nepali "।" marks a completion of a declarative sentence. So can we keep "।" by removing the "." instead?

Yeah, I remember the symbol. I was confused since there were 2 periods It seems like you fixed it.

We keep native symbols for periods for page descriptions. So we can use `।` here.